e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -77,6 +77,9 @@ function setNotificationSettings(desiredSettings = {first: true, username: true,
 
     // Verify that we now have a Notification property
     cy.window().should('have.property', 'Notification');
+
+    // # Navigate to a channel we are NOT going to post to
+    cy.get('#sidebarItem_saepe-5').scrollIntoView().click({force: true});
```

Comment:
At-mention didn't work since the user is on the town-square channel already. Go to `saepe-5` channel after calling `cy.toAccountSettingsModal` where under the hood it visits town-square channel.